### PR TITLE
Add 1.4 release blog

### PR DESCRIPTION
**Pull request by youngnick**
Signed-off-by: Nick Young <ynick@vmware.com>

### CODE REVIEW

File: site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Contour now has the ability to write a `status.loadBalancer.addresses` block to Ingress objects. This block is used by services which need to know how to reach an Ingress' backing service from outside the cluster, like [external-dns](https://github.com/kubernetes-sigs/external-dns).
+
+There are two ways for Contour to find this information:
+- by watching a Service object for the Envoy service, and putting the associated `status.loadBalancer` block from that Service into all associated Ingress objects. This is what is used in the example deployment.
+- Operators can also specify an address on Contour's command line, using the `--ingress-status-address` flag. The address that’s passed on the command line will be passed straight through to the Ingress status.
+
+This also means that when you `kubectl get` a Contour-owned Ingress, instead of this:
+
+```
+➜ kubectl get ingress httpbin
```
Comment (jpeach) [inline site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md:69]: Nit: We use `$` as a shell prompt.

File: site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Future Plans
+
+The Contour project is very community-driven and the team would love to hear your feedback! 
+
+- Come talk about topics at our next community meeting.
+- We’ve heard that a number of teams have forked Contour and we would love to hear about what changes you needed, and to see if we can help to bring them upstream.
+Please consider coming to our community meeting, or contact us: either via an issue, or hit me up on Twitter [@youngnick](https://twitter.com/youngnick).
+
+If you are interested in contributing, a great place to start is to comment on one of the issues labeled with [Help Wanted](https://github.com/heptio/contour/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) and work with the team on how to resolve them.
```
Comment (jpeach) [inline site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md:94]: Heptio?

Comment (youngnick) [reply site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md:94]: 🤦

**Comment by stevesloka** [comment]
A couple of nit errors in the site proofing step of the CI job that failed.

File: site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+## Are you a Contour user? We would love to know!
+If you're using Contour and want to add your organization to our adopters list, please visit this [page](https://github.com/projectcontour/contour/blob/master/ADOPTERS.md).
+If you prefer to keep your organization name anonymous but still give us feedback into your usage and scenarios for Contour, please post on this [GitHub thread](https://github.com/projectcontour/contour/issues/1269)          
+
+## Join the Contour Community!
+- Join the Contour Community Meetings via the instructions on our [Community Page](https://projectcontour.io/community/)
+- Get updates on Twitter ({{site.footer_social_links.Twitter.url}})
+- Chat with us on Slack ({{site.footer_social_links.Slack.url}})
+- Collaborate with us on GitHub: {{site.github.repository_url}}
```
Comment (jonasrosland) [inline site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md:104]: This is covered in a community footer for each blog so you don't have to add it: https://github.com/projectcontour/contour/blob/master/site/_includes/blog-community-footer.html

File: site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+We’re immensely grateful for all the community contributions that help make Contour even better! Special thanks go out to:
+- Tero Saarni ([@tsaarni](https://github.com/tsaarni))
+- Peter Grant ([@pickledrick](https://github.com/pickledrick))
```
Comment (jonasrosland) [inline site/_posts/2020-04-27-client-cert-auth-ingress-improvements.md:108]: Add a "Thank you to our contributors" or similar title to this.

**Comment by youngnick** [comment]
The two links that are showing as broken in one of the builds are the ones for this blog post, and the one for my tag, which has no blogs in it until this is published.

**Comment by youngnick** [comment]
@jonasrosland All done from what I can see. Feel free to merge this and promote.

**Pull request merged**